saltboot/salt.go

```diff
@@ -363,6 +354,47 @@ func SaltServerStopRequestHandler(w http.ResponseWriter, req *http.Request) {
 	resp.WriteHttp(w)
 }
 
+func SaltServerChangePasswordHandler(w http.ResponseWriter, req *http.Request) {
+	log.Println("[SaltServerChangePasswordHandler] execute salt master change password request")
+
+	decoder := json.NewDecoder(req.Body)
+	var saltActionRequest SaltActionRequest
+	if err := decoder.Decode(&saltActionRequest); err != nil {
+		log.Printf("[SaltServerChangePasswordHandler] [ERROR] couldn't decode json: %s", err.Error())
```

Comment:
I think we should not log err.Error() as it might contains the content of the request body with the password.